app/views/admin/users/show.html.erb

```diff
@@ -1,50 +1,112 @@
-<div class="grid gap-6">
-  <div class="crayons-card p-6">
-    <h2 class="d-inline">
+<%# this is for pawel's testing purpose to emulate tabs... %>
+<% current_tab = "overview" # overview|emails|notes|reports|flags %>
```

Comment:
FYI: this is how I test individual tabs locally since we don't have views for each yet.